src/Gui/PreferencePages/DlgSettingsEditor.cpp

```diff
@@ -301,15 +303,21 @@ void DlgSettingsEditor::loadSettings()
     QStringList fixedFamilyNames;
     for (const auto &name : familyNames) {
         if (QFontDatabase().isFixedPitch(name)) {
-            fixedFamilyNames.append(name);
+            std::string utf8_name = name.toUtf8().constData();
+            if (boost::iequals(utf8_name, "8514oem")) {
```

Comment:
This filters out everything but 8514oem. So, it does the opposite. And since we already use QString I don't see the necessity to switch to another lib for string comparison.

IMO, the easiest solution would be to remove "8514oem" after the string list has been filled:
`fixedFamilyNames.removeAll(QString::fromLatin1("8514oem"));`

Reply:
Thanks Werner, I obviously need some sleep. I'm not relying on the font definitively staying in that case so prefer to have case insensitivity.